pkg/action/add_bundle_to_configs.go

```diff
@@ -0,0 +1,83 @@
+package action
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/otiai10/copy"
+	"github.com/sirupsen/logrus"
+
+	"github.com/operator-framework/operator-registry/internal/declcfg"
+	"github.com/operator-framework/operator-registry/pkg/registry"
+)
+
+type AddConfigRequest struct {
+	ConfigsDir string
+	Bundles    []BundleExtractor
+}
+
+type BundleAdder struct {
+	Logger *logrus.Entry
+}
+
+func NewBundleAdder(logger *logrus.Entry) BundleAdder {
+	return BundleAdder{
+		Logger: logger,
+	}
+}
+
+func (b BundleAdder) AddToConfig(request AddConfigRequest) error {
```

Comment:
A general comment on code formatting for all functions in this PR. You should add an empty line after each block of `{}` to make the code a bit more spacing-out so it will be easier to read and review. Line after line of code is too dense.

Reply:
Do you mean empty line at line 30 (which is there right now) or at line 32? 
In a previous PR when I introduced an empty line in similar lines like 32, I've been told the preferred approach for this repo is to not include an empty line. 

I prefer the empty line at 32 too, but as long as we can agree with one convention and stick to it I'm fine with either 🤷🏽‍♂️

Reply:
No. I mean after a block of code `{}`, NOT right after the func signature. For example, at like 36, there should be an empty line afterwards. You can look at [this code](https://github.com/cuelang/cue/blob/master/cuego/cuego.go#L131) for example on how they have empty lines in between block of code to improve readability.

Reply:
Done!